### ISSUE TITLE
Missing assert for when no primary key is defined during insert

### ISSUE THREAD

**Issue by mrdavidrees** [simolus3/moor#19]
A table defined like below, will throw an sql error when it attempts to add an item with a empty primary key, however when deleting an item, it will correctly assert that the table is missing a primary key

```
class MyTable extends Table {
  IntColumn get id => integer()();
}
```

```
//throws assert error correctly
delete(myTable).delete(item);
```

```
//does not throw assert error correctly
into(myTable).insert(item);
```

Error thrown:
```
DB Error: 1 "near ")": syntax error"
DB Query: CREATE TABLE IF NOT EXISTS my_table (id INTEGER NOT NULL, PRIMARY KEY ());
```

**Comment by simolus3** [comment]
Thanks for the report!

It seems like there is another problem here, which is that tables without a primary key don't create a valid `CREATE TABLE` command.

**Comment by simolus3** [comment]
I just published version `1.3.1` of `moor_generator` and `moor`, which fixes this problem.
If you're already using `1.3`, upgrading your flutter packages should resolve the problem.
